functions/ts/kubeval/README.md

```diff
@@ -0,0 +1,26 @@
+# kubeval
+
+### Overview
+
+Validates configuration using [kubeval].
+
+### Synopsis
+
+If neither `schema_location` nor `additional_schema_locations` is provided, we
```

Comment:
The Synopsis is start off in the weeds. Please reorder and expand. How does kubeval work in 1 or 2 sentences, explain it needs OpenAPI document, etc.